frontend/src/components/simulation/sidebar/TramSidebarComponent.vue

```diff
@@ -115,6 +115,13 @@ watch(
       </div>
       <div class="value">{{ tramDetails?.speed }} km/h</div>
     </div>
+    <div class="section">
```

Comment:
Please add some spacing here:
```suggestion

    <div class="section">
```